pkg/kv/kvserver/client_replica_circuit_breaker_test.go

```diff
@@ -461,8 +456,8 @@ func TestReplicaCircuitBreaker_RangeFeed(t *testing.T) {
 	require.NoError(t, tc.RestartServer(n2))
 
 	tc.UntripsSoon(t, tc.Write, n1)
-	readOneVal(t, stream1)
-	readOneVal(t, stream2)
+	readOneVal(t, ctx, stream1)
+	readOneVal(t, ctx, stream2)
```

Comment:
For some reason this times out. The read for stream1 above succeeds:

```
client_replica_circuit_breaker_test.go:459: val:<key:"\372" value:<raw_bytes:"\344\0316\222\003hello-4" timestamp:<wall_time:1645705031776298000 > > prev_value:<timestamp:<> > > 
```

but stream2 just gets checkpoint events. I've tried adding a `time.Sleep(5*time.Second); tc.Write(n1)` but it doesn't change anything. What's going on with this stream? It should be making progress I'd think.

```
    client_replica_circuit_breaker_test.go:376: rangefeed1: ignoring event: checkpoint:<span:<key:"\372" end_key:"\377\377" > resolved_ts:<wall_time:1645705028776998000 > > 
    client_replica_circuit_breaker_test.go:376: rangefeed1: ignoring event: checkpoint:<span:<key:"\372" end_key:"\377\377" > resolved_ts:<wall_time:1645705028856069000 > > 
    client_replica_circuit_breaker_test.go:376: rangefeed1: ignoring event: checkpoint:<span:<key:"\372" end_key:"\377\377" > resolved_ts:<wall_time:1645705029121107000 > > 
```